oracle.go

```diff
@@ -65,6 +83,18 @@ func (o *Oracle) Initialize(ctx context.Context, req dbplugin.InitializeRequest)
 		usernameTemplate = defaultUsernameTemplate
 	}
 
+	splitStatements, err := coerceToBool(req.Config, "split_statements", true)
+	if err != nil {
+		return dbplugin.InitializeResponse{}, fmt.Errorf("failed to parse 'do_not_split_queries' field: %w", err)
```

Comment:
Would it be too restrictive to just not allow anonymous PL/SQL blocks in the revocation statement list? That would eliminate the need for this parameter.

Reply:
@vt-ms Unfortunately that wouldn't eliminate the need for this parameter because it's allowing for the current behavior of taking a single string `CREATE USER ...; ALTER USER ...; FOO BAR ...;` and splitting it into several strings before interpreting them as SQL statements (`CREATE USER...` `ALTER USER ...` `FOO BAR ...`). The current behavior prevents users from doing more complex queries (such as anonymous PL/SQL blocks). If we removed it and stopped splitting those statements, current Vault users would break.